forch/port_state_manager.py

```diff
@@ -340,6 +366,13 @@ def _handle_session_result(self, mac, session_result):
     def _handle_unauthenticated_state(self, mac):
         self._update_device_state_varz(mac, DVAState.unauthenticated)
 
+    def _handle_scheduled_sequstering(self, mac):
```

Comment:
I think there might be an issue here where multiple timers can get set for the same mac, and then the first one will fire even if the date was changed to later?   Likely doesn't matter, but maybe just add a check here to only actually do the sequester stuff once.  Or at least something to consider.  At the very least add a comment or note saying that this is a known "problem" in the code.